ui/src/types/dashboards.ts

```diff
@@ -199,3 +199,19 @@ export type NewDefaultCell = Pick<
   Cell,
   Exclude<keyof Cell, 'i' | 'axes' | 'colors' | 'links' | 'legend'>
 >
+
+export interface DashboardMetadata {
+  name: string
+  icon: string
+  version: string
+  dashboardVersion: string
+  description: string
+  author: string
+  license: string
+  url: string
+}
+export interface Protoboard {
+  id: string
+  meta: DashboardMetadata
```

Comment:
Could we change the name of this to `meta:ProtoboardMeta`, and `data: ProtoboardData`?
In the work we did with Jared, we were just trying to distinguish the concepts of Dashboards and Protoboards as much as possible since they are already so close. 
 ProtoboardData can just be {cells:object}

Reply:
Updated